### PR TITLE
[13.0][ADD] account_move_date_maturity

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
Adds date_maturity field with Journal Entries action

### CODE REVIEW

File: account_invoice_date_maturity/views/account_move_views.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="account.action_move_journal_line" model="ir.actions.act_window">
+        <field name="context">{'default_type': 'entry', 'search_default_misc_filter':1}</field>
```
Comment (dalonsod) [inline account_invoice_date_maturity/views/account_move_views.xml]: I'd emphasize that change in this way (setting value to `False`, and documenting original context):
```suggestion
        <!-- Overriden context: {'default_type': 'entry', 'search_default_misc_filter':1, 'view_no_maturity': True} -->
        <field name="context">{'default_type': 'entry', 'search_default_misc_filter':1, 'view_no_maturity': False}</field>
```

File: account_invoice_date_maturity/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Account Invoice Date Maturity",
+    "summary": """
+        Adds date_maturity field with Journal Entries action
```
Comment (dalonsod) [inline account_invoice_date_maturity/__manifest__.py]: Better this (human readable field name, and right model name):
```suggestion
        Makes Due Date visible for Journal Items
```

File: account_invoice_date_maturity/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "13.0.1.0.0",
+    "category": "Account",
+    "website": "https://github.com/solvosci/slv-account",
+    "depends": [
+        "account",
+    ],
+    "data": [
+        "views/account_move_views.xml",
```
Comment (dalonsod) [inline account_invoice_date_maturity/__manifest__.py]: Action redefined in this file belongs to `account.move.line` model, so this file should be renamed.

File: account_invoice_date_maturity/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Account Invoice Date Maturity",
```
Comment (dalonsod) [inline account_invoice_date_maturity/__manifest__.py]: Better this:
```suggestion
    "name": "Journal Item Due Date visible",
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comments... and I unhappily have to say that I don't agree with module name,,, better `account_move_line_date_maturity` , as it places in the right model. Sorry.

When comments are attended, please install it in AluTest environment.

cc @Anxo82

File: account_invoice_date_maturity/__manifest__.py (hunk 3 above)
Comment (dalonsod) [reply account_invoice_date_maturity/__manifest__.py]: Oops I was wrong, such action belongs to `account.move`. Forget it.

**Comment by dalonsod** [comment]
> See comments... and I unhappily have to say that I don't agree with module name,,, better account_move_line_date_maturity , as it places in the right model. Sorry.

Nope! `account_move_date_maturity` , not `line`, but better than `invoice`.

**Comment by ChristianSantamaria** [comment]
Instaled in Test Enviroment

**Comment by dalonsod** [comment]
@lmiguens-solvos already installed in AluTest environment, could you check?

**Comment by lmiguens-solvos** [comment]
LGTM, visto que ya sale la opción de seleccionar la fecha de vencimiento donde antes no salía
image

**Review by dalonsod — APPROVED** [review]
:+1:

**Pull request merged**